eq-author/src/index.js

```diff
@@ -84,13 +86,15 @@ const headersLink = setContext((_, { headers }) =>
 );
 
 const link = ApolloLink.from([
-  createErrorLink(store),
+  createErrorLink(getStore),
   headersLink,
   networkLink,
 ]);
 
 const client = createApolloClient(link, cache);
 
+store = configureStore(history, client);
```

Comment:
`history` and `client` are not being used by `configureStore`, can they be removed?